src/blocks/shared/components/error-hint.tsx

```diff
@@ -0,0 +1,20 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+export const AD_BLOCKER_HINT: string = __(
+	'This error can be sometimes caused by ad-blockers or browser tracking protections. Please whitelist this website where needed and try again.',
+	'wp-parsely',
+);
+
+export function ErrorHint(): JSX.Element {
```

Comment:
Yes but I prefer to have types ... it seems easy for me to read the code and know the type 🙂

Reply:
Makes sense, mind the `null` case

Reply:
In other words, could this fail?

Reply:
Not as written here. But a component that sometimes `return`s null would need its return type adjusted:

```
function ConditionalComponent ( props: Props ): JSX.Element | null {
  if ( props.foo ) {
    return null;
  }

  return <div>Hello</div>;
}
```

This is nothing surprising, it's just something you have to pay attention to in TypeScript. But sometimes React-provided types feel a bit mysterious, especially surrounding JSX. I personally think explicit typing is always fine, but it can sometimes get burdensome to the point where you start looking around for abstractions. React component return types should always be pretty simple, though.